llvm/test/LTO/X86/empty-triple.ll

```diff
@@ -0,0 +1,7 @@
+; RUN: llvm-as < %s >%t1
+; RUN: llvm-lto -o %t2 %t1  2>&1 | FileCheck --allow-empty %s
+
+; The test is to check the triple is set to default one when it's empty.
+; Otherwise, an error will be raised by llvm-lto.
+
+; CHECK-NOT: llvm-lto: error: No available targets are compatible with triple ""
```

Comment:
```suggestion
; CHECK-NOT: error
```

Reply:
Alternatively there's not much gained by checking stderr, since it's not exiting 1 anymore. If you want to check something, that the output is valid might be better